configs/crypto-config-symmetric-only.h

```diff
@@ -1,91 +0,0 @@
-/**
```

Comment:
Comment not related to this file: by the way could you fix the second line of `crypto-config-suite-b.h`, currently `crypto-config-symmetric-only.h` instead of `crypto-config-suite-b.h`.

Reply:
Resolved.